CHANGELOG.md

```diff
@@ -6,3 +6,4 @@ This project adheres to [Semantic Versioning](http://semver.org/).
 ### Fixed
 - Added missing default sort by primary key(s).
 - Allow non integer/string types as identifier (ex. uuid).
+- A column has been specified more than once in the order by list
```

Comment:
What should I write in body? This:  Fix issue #565 "A column has been specified more than once in the order by list" alredy writed. What else?